7.6.break,_continue_и_else/7.6.8.py

```diff
@@ -0,0 +1,5 @@
+n = int(input())
+for i in range(1, n + 1):
+    if i in range(5, 10) or i in range(17, 38) or i in range(78, 88):
+        continue      
+    print(i)
```

Comment:
Молодец, но лучше не использовать в этом случае функцию range(), она более трудоемкая, чем проверка на неравенство:
```suggestion
for i in range(1, n + 1):
    if 5 <= n <= 9 or 17 <= n <= 37 or 78 <= n <= 87:
        continue      
    print(i)
```